pyproject.toml

```diff
@@ -11,10 +11,11 @@ classifiers = [
   'Operating System :: MacOS',
   'Operating System :: Microsoft :: Windows',
   'Operating System :: POSIX',
+  'Programming Language :: Python :: 3.9',
```

Comment:
Although I agree with you on the sorting here, it is currently required to be in the original format. See https://github.com/pyvista/pyvista/pull/6781#issuecomment-2691508280